packages/core/src/routers/SwitchRouter.js

```diff
@@ -71,13 +71,13 @@ export default (routeConfigs, config = {}) => {
         ...childRouter.getStateForAction(childAction),
         key: routeName,
         routeName,
-        params,
+        ...(params ? { params } : {}),
       };
     }
     return {
       key: routeName,
       routeName,
-      params,
+      ...(params ? { params } : {}),
```

Comment:
This doesn't seem to change any behaviour and seems to be more code. Can you confirm?

Reply:
Since `params` can be `null` this will avoid having the property defined! It's how it's done in StackRouter IIRC :)